descriptions/api.github.com/api.github.com.yaml

```diff
@@ -18157,6 +18233,20 @@ paths:
           type: string
       - "$ref": "#/components/parameters/per_page"
       - "$ref": "#/components/parameters/page"
+      - name: top
+        description: legacy parameter for pagination.
+        in: query
+        deprecated: true
+        required: false
+        schema:
+          type: string
+      - name: last_sha
+        description: legacy parameter for pagination.
+        in: query
+        deprecated: true
+        required: false
+        schema:
+          type: string
```

Comment:
I'd suggest not to publicly document legacy parameters. They will find their way into the SDKs through code generation and instead of fewer people using them, it will be more.

Reply:
That's a good call. We want to document legacy parameters when they have been documented before, but not the ones that were never described since the existence of this document. I'll open a PR to filter those with a special attribute.